source/includes/k8s/deploy-operator.rst

```diff
@@ -64,20 +63,14 @@ The tenant utilizes Persistent Volume Claims to talk to the Persistent Volumes t
 Prerequisites
```

Comment:
The right hand nav right now has:

- Prerequisites
- Procedure
- Overview
- Prerequisites
- Procedure

Need to do a bit of compacting to clean that nav up.

Reply:
Fixing more include nonsense 🙃